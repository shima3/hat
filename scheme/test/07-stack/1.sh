#!/bin/bash
$* -e main examples/stack.sch
