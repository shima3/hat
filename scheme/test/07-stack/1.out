main 1 flag=(^ (then else) #t else then)
main 2 value=(^ (a) b)
main 3 flag=(^ (then else) #t else then)
main 4 value=(^ (a) b)
main 5 flag=(^ (then else) #t else then)
main 6 value=a
main 7 flag=(^ (then else) #f else then)
